apisix/cli/ngx_tpl.lua

```diff
@@ -229,6 +229,10 @@ http {
     real_ip_header {* http.real_ip_header *};
     {% end %}
 
+    {% if http.real_ip_recursive then %}
+    real_ip_recursive {* http.real_ip_recursive *};
```

Comment:
Need to add a test like: https://github.com/apache/apisix/blob/9a40fd7cb497974f9d7bd75f128eed5dd238cdc6/t/cli/test_main.sh#L382-L399